package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "homepage": "https://kerrimov.github.io/the-ssspace",
   "dependencies": {
+    "@craco/craco": "^6.4.3",
```

Comment:
why do you add craco?

Reply:
because we use create react app lib. In this case webpack config is hidden. To change it we need to use some extra lib (for example, craco, react-app-rewired etc).

Reply:
Can be used absolute imports as an option. Discussed.